src/Utils/Hexagons.svelte

```diff
@@ -344,83 +386,136 @@ const sleep = (milliseconds) => {
-        on:mouseup={e => handleTouchEnd(e)}
-        on:touchstart={e => handleTouchStart(e)}
-        on:touchmove={e => handleTouchMove(e)}
-        on:touchend={e => handleTouchEnd(e)}
-        >
-
-        {#each drawableHexagons as hexagon}
-            <g key={`g-${hexagon.id}`}>
-                <polygon
-                    id={`hex-${hexagon.id}`}
-                    key={`hex-${hexagon.x}-${hexagon.y}`}
-                    class="activatable"
-                    fill="none"
-                    stroke={hexagon.color}
-                    stroke-width="5px"
-                    strokeLinejoin="miter"
-                    transform={`translate(${hexagon.x - (hexagon.width / 2)} ${hexagon.y - (hexagon.height / 2)})`}
-                    points={hexagon.points} />
-                    <text id={`text-${hexagon.id}`} x={hexagon.x - 6} y={hexagon.y}>{hexagon.id}</text>
-                </g>
-        {/each}
-    </svg>
-    {#if test_ui}
-        {#each hexagons as actuator}
-            <button on:click={buildCommandBlocks(actuator.id)}> Button {actuator.id}</button>
-        {/each}
-        <p>Test: {$block0_31}, {$block32_63}, {$block64_95}, {$block96_127}</p>
-
+<div class="full-container" style={setBackground}>
+    {#if arraySize == "normal"}
+        <div class="col-5">
+            <select bind:value={arrayType}>
+                {#each hexagonTypes as type}
+                    <option value={type.id}>
+                        {type.text}
+                    </option>
+                {/each}
+            </select>
+            <br/>
+            <button on:click={() => moveable.request("rotatable",{deltaRotate: -(rotation)}, true)}>Rotate {rotation}&#730 &#8634</button>    
+            <button on:click={() => moveable.request("rotatable",{deltaRotate: +(rotation)}, true)}>Rotate {rotation}&#730 &#8635</button>    
+            <input style="width: 50%" bind:value={rotation}/> &deg
+            <br/>
+            <button on:click={() => moveable.request("rotatable",{rotate: initialRotation}, true)}>Reset</button>
+        </div>
     {/if}
+
+    <div class={className}>
+        <svg  version="1.0" xmlns="http://www.w3.org/2000/svg" bind:this={target} style={setStyle}
+            width={`${viewBox.x}px`}
+            height={`${viewBox.y}px`}
+            viewBox={`0 0 ${viewBox.x} ${viewBox.y}`}
+            preserveAspectRatio="xMidYMid meet"
+            on:mousedown={e => handleTouchStart(e)}
+            on:mousemove={e => handleTouchMove(e)}
+            on:mouseup={e => handleTouchEnd(e)}    
+            >
+            {#each drawableHexagons as hexagon}
+                <g key={`g-${hexagon.id}`}>
+                    <polygon
+                        id={`hex-${hexagon.id}`}
+                        key={`hex-${hexagon.x}-${hexagon.y}`}
+                        class="activatable"
+                        fill="none"
+                        stroke={hexagon.color}
+                        stroke-width={strokeWidth}
```

Comment:
Does this property actual get used? We had to change to camel case for stroke join.

Reply:
The camel case and dash for the properties does seem inconsistent, but stroke-width was what actually worked.